src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -47,17 +48,25 @@ const SubsidiesSummary = ({ className, showSearchCoursesCta }) => {
     request => request.state === SUBSIDY_REQUEST_STATE.PENDING_REVIEW,
   ), [couponCodeRequests.length]);
 
-  if (!(hasActiveSubsidies || pendingCouponCodeRequests.length)) {
+  const pendingLicenseRequest = useMemo(() => licenseRequests.find(
+    request => request.state === SUBSIDY_REQUEST_STATE.PENDING_REVIEW,
```

Comment:
Looking at this `PENDING_REVIEW` state should it be just `PENDING` to match the work done in https://github.com/edx/enterprise-access/pull/15?

Reply:
updated